quickwit/quickwit-actors/src/universe.rs

```diff
@@ -99,11 +101,24 @@ impl Universe {
     /// `tokio::time::sleep`.
     ///
     /// It can however be accelerated when using a time-accelerated
-    /// Universe.
+    /// universe.
     pub async fn sleep(&self, duration: Duration) {
         self.spawn_ctx.scheduler_client.sleep(duration).await;
     }
 
+    /// This function acts as a drop-in replacement of
+    /// `tokio::time::sleep`.
+    ///
+    /// It can however be accelerated when using a time-accelerated
+    /// universe.
+    pub async fn timeout<O>(
```

Comment:
I added this to fix the unit tests for my first attempt but did not use it for the second. I chose to leave it.